index.js

```diff
@@ -46,21 +42,21 @@ app.post('/create', function (req, res) {
 })
 
 app.post('/auth', (req, res) => {
-    db.collection('Users').find({}).toArray(function (err, result) {
-        var x = 0;
+
+    db.collection('Users').findOne({ "username": req.body.username}, function (err, result) {
         if (err) throw err;
-        for (var i = 0; i < result.length; i++) {
-            if (req.body.username == result[i].username && req.body.password == result[i].password) {
-                var y = result[i].username
-                var z = result[i].password
-                x = 1;
+        if (result != null) {
+            if (req.body.password == result.password) {
+                req.session.loggedIn=true;
+                req.session.username=result.username;
+                if (result.username == "yashpal") {
+                    res.redirect('/topics')
+                } else {
+                    res.redirect('/topics')
+                }
             }
-        }
-        if (x == 1) {
-            if (y == "yashpal") {
-                res.redirect('/topics')
-            } else {
-                res.redirect('/topics')
+            else {
+                res.send(`<script>alert('wrong credentials!');window.location='/'</script>`);
```

Comment:
Remove alert message if user is successfully logged in redirect him to topic page if user is not successfully logged in
then create message object like
let data ={}
if(user is not successfully logged in ){
data.status = false;
}else{
data.status = true;
}
response.render("your_hbs_page.hbs",data)